test.sh

```diff
@@ -20,7 +20,7 @@ cd ..
 
 # Be more strict with HTTP push tests: run with race detector enabled
 cd server
-go test -run Push_ -race
+go test -run Push_
```

Comment:
```suggestion
```
I think we can remove it at all. This test is run anyway without `-race` in line 6: `go test -coverprofile cover.out $(go list ./... | grep -v 'test/e2e')`